robot/server/server.go

```diff
@@ -404,3 +406,30 @@ func (s *Server) SendSessionHeartbeat(ctx context.Context, req *pb.SendSessionHe
 	}
 	return &pb.SendSessionHeartbeatResponse{}, nil
 }
+
+func (s *Server) ModuleLog(ctx context.Context, req *pb.ModuleLogRequest) (*pb.ModuleLogResponse, error) {
+	log := req.Log
+	if log == nil {
+		return nil, errors.New("ModuleLogRequest received with no associated Log")
+	}
+
+	// Use a sublogger of robot logger with module's logger name. Disable caller
+	// to mimic caller passed in from module.
+	logger := s.r.Logger().Sublogger(log.LoggerName).WithOptions(zap.WithCaller(false))
```

Comment:
@dgottlieb even with my change of passing `inUTC` through to subloggers, the `logger` created here will still be in EST (local time) because `WithOptions` converts to an actual zap logger and we lose the UTCness of the sublogger 😢. Not sure what to do about that, but reining in all our log timezones is probably out of scope for this change anyway?

Reply:
Yep! Noted in RSDK-6280 and put a `TODO` to preserve timezones above `moduleLogger`'s declaration.